tools/docker/docker-compose-tls.yaml

```diff
@@ -0,0 +1,196 @@
+    build:
+      context: ../../
+      dockerfile: tools/docker/Dockerfile
+    environment: *common-env
+    command:
+      - /bin/bash
+      - -c
+      - >-
+        aap-eda-manage migrate
+        && aap-eda-manage create_initial_data
+        && scripts/create_superuser.sh
+        && aap-eda-manage runserver 0.0.0.0:8000
+    ports:
+      - '8000:8000'
+    depends_on:
+      redis:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+    healthcheck:
+      test: [ 'CMD', 'curl', '-q', 'http://localhost:8000/_healthz' ]
+      interval: 30s
+      timeout: 5s
+      retries: 10
+    volumes:
+      - '../../:/app/src:z'
+
+  eda-ws:
+    image: "${EDA_IMAGE:-localhost/aap-eda}"
+    environment: *common-env
+    command:
+      - /bin/bash
+      - -c
+      - >-
+        aap-eda-manage runserver 0.0.0.0:8000
+    ports:
+      - '8001:8000'
+    depends_on:
+      eda-api:
+        condition: service_healthy
+    volumes:
+      - '../../:/app/src:z'
+
+
+  eda-scheduler:
+    image: "${EDA_IMAGE:-localhost/aap-eda}"
+    environment: *common-env
+    command:
+      - /bin/bash
+      - -c
+      - >-
+        aap-eda-manage scheduler
+    depends_on:
+      redis:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+    volumes:
+      - '../../:/app/src:z'
+
+  eda-default-worker:
+    deploy:
+      replicas: ${EDA_DEFAULT_WORKERS:-1}
+    image: "${EDA_IMAGE:-localhost/aap-eda}"
+    environment: *common-env
+    command:
+      - aap-eda-manage
+      - rqworker
+      - --worker-class
+      - aap_eda.core.tasking.DefaultWorker
+    depends_on:
+      eda-api:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+    volumes:
+      - '../../:/app/src:z'
+    restart: always
+
+  eda-activation-worker:
+    deploy:
+      replicas: ${EDA_ACTIVATION_WORKERS:-2}
+    image: "${EDA_IMAGE:-localhost/aap-eda}"
+    command:
+      - aap-eda-manage
+      - rqworker
+      - --worker-class
+      - aap_eda.core.tasking.ActivationWorker
+    environment: *common-env
+    depends_on:
+      eda-api:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+    volumes:
+      - '../../:/app/src:z'
+    restart: always
+
+  postgres:
+    image: 'quay.io/sclorg/postgresql-13-c9s:latest'
+    environment:
+      POSTGRESQL_USER: eda
+      POSTGRESQL_PASSWORD: secret
+      POSTGRESQL_ADMIN_PASSWORD: secret
+      POSTGRESQL_DATABASE: eda
+      POSTGRES_HOST_AUTH_METHOD: trust
+    ports:
+      - '5432:5432'
+    volumes:
+      - 'postgres_data:/var/lib/pgsql/data'
+      - "${EDA_PGSSL_PATHCONFIG:-./postgres_ssl_config/postgresql-cfg}:/opt/app-root/src/postgresql-cfg:Z"
+      - "${EDA_PGSSL_PATHCERT:-./postgres_ssl_config/certs}:/opt/app-root/src/certs:Z"
```

Comment:
I think this must be a lowercase "z"

Reply:
we already have the ":z" in the source code mountpoint without no issues. I suggest to do the same.

Reply:
Sure!  let me update this.